docs/user_guide/rcm.md

```diff
@@ -4,7 +4,15 @@ RCM (short for ReCovery Mode) is a mode for the switch that allows nintendo to s
 
 -----
 
-### Emunand cfw (Recommended)
+!!!tip "Emunand vs Sysnand"
+	- Emunand stands for Emulated nand, this means that your entire nand 	   (system memory) will run off your sd card with custom firmware (CFW)     this nand is *completely seprate* from your sysnand.
+
+	- Sysnand stands for System nand, meaning the internal storage in your actual switch, which means that cfw will run off your nand that is on your actual system, which presents a higher ban risk.
```

Comment:
Should be `sysNAND` and `NAND`. CFW should also be capitalized.

Reply:
Nand format unchanged from original guide.